### PR TITLE
revamp README

### PR DESCRIPTION
**Pull request by zeke**
This PR updates the README:

- Puts content in the chronological order that integrators need it: Installation, environment setup, usage...
- Adds headings so we can deep link to specific parts of the README
- Replaces the resnet image input example with [something more relevant](https://replicate.com/methexis-inc/img2prompt)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,66 @@
...
-Grab your token from [replicate.com/account](https://replicate.com/account) and authenticate by setting it as an environment variable:
+## Install
+
+```sh
+pip install replicate
+```
+
+## Authenticate
+
+Before running any Python scripts that use the API, you need to set your Replicate API token in your enveronment.
```
Comment (erbridge) [inline README.md:13]:
```suggestion
Before running any Python scripts that use the API, you need to set your Replicate API token in your environment.
```

File: README.md [hunk 2]
```diff
@@ -1,44 +1,66 @@
...
+import replicate
+model = replicate.models.get("stability-ai/stable-diffusion")
+version = model.versions.get("27b93a2413e7f36cd83da926f3656280b2931564ff050bf9575f1fdf9bcd7478")
+version.predict(prompt="a 19th century portrait of a wombat gentleman")
+
+# ['https://replicate.com/api/models/stability-ai/stable-diffusion/files/50fcac81-865d-499e-81ac-49de0cb79264/out-0.png']
 ```
 
-Some models, like [replicate/resnet](/replicate/resnet), receive images as inputs. To pass a file as an input, use a file handle or URL
+Some models, like [methexis-inc/img2prompt](https://replicate.com/methexis-inc/img2prompt), receive images as inputs. To pass a file as an input, use a file handle or URL
```
Comment (erbridge) [inline README.md:36]:
```suggestion
Some models, like [methexis-inc/img2prompt](https://replicate.com/methexis-inc/img2prompt), receive images as inputs. To pass a file as an input, use a file handle or URL:
```

**Review by erbridge — APPROVED** [review]
Looks great. Couple of minor edits.

File: README.md (hunk 1 above)
Comment (zeke) [reply README.md:13]: enveronment haha

**Pull request merged**